docs/attributes-registry/cloudfoundry.md

```diff
@@ -0,0 +1,90 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: CloudFoundry
+--->
+
+# CloudFoundry
+
+## CloudFoundry Resource Attributes
+
+<!-- semconv registry.cloudfoundry(omit_requirement_level) -->
+| Attribute  | Type | Description  | Examples  |
+|---|---|---|---|
+| `cloudfoundry.app.id` | string | The guid of the application. [1] | `218fc5a9-a5f1-4b54-aa05-46717d0ab26d` |
+| `cloudfoundry.app.name` | string | The name of the application. [2] | `my-app-name` |
+| `cloudfoundry.instance_id` | string | An index or guid describing the concrete instance of the event source. [3] | `0`; `218fc5a9-a5f1-4b54-aa05-46717d0ab26d` |
+| `cloudfoundry.org.id` | string | The guid of the CloudFoundry org the application is running in. [4] | `218fc5a9-a5f1-4b54-aa05-46717d0ab26d` |
+| `cloudfoundry.org.name` | string | The name of the CloudFoundry organization the app is running in. [5] | `my-org-name` |
+| `cloudfoundry.process.id` | string | The UID identifying the process. [6] | `218fc5a9-a5f1-4b54-aa05-46717d0ab26d` |
+| `cloudfoundry.process.type` | string | The type of process. [7] | `web` |
+| `cloudfoundry.source_id` | string | A guid or another name describing the event source. [8] | `218fc5a9-a5f1-4b54-aa05-46717d0ab26d`; `cf/gorouter` |
```

Comment:
If there is a `source_id` and a `source_type`, shouldn't the naming rather be: `source.id` and `source.type`, i.e. `source` being a grouping prefix?